crates/ruff_linter/src/rules/flake8_boolean_trap/rules/boolean_positional_value_in_call.rs

```diff
@@ -52,6 +52,10 @@ impl Violation for BooleanPositionalValueInCall {
 }
 
 pub(crate) fn boolean_positional_value_in_call(checker: &mut Checker, call: &ast::ExprCall) {
+    // https://github.com/astral-sh/ruff/issues/14535
+    if checker.source_type.is_stub() {
+        return;
+    }
```

Comment:
This one I'm not sure about. The other two FBT rules are about definitions, which a stub author doesn't have control over if faithfully representing the runtime implementation.

But this rule is about calls, which shouldn't be found in stubs in the first place (maybe in decorators? In which case it could be argued to apply)